Index: .idea/misc.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+><?xml version=\"1.0\" encoding=\"UTF-8\"?>\r\n<project version=\"4\">\r\n  <component name=\"ExternalStorageConfigurationManager\" enabled=\"true\" />\r\n  <component name=\"MavenProjectsManager\">\r\n    <option name=\"originalFiles\">\r\n      <list>\r\n        <option value=\"$PROJECT_DIR$/pom.xml\" />\r\n      </list>\r\n    </option>\r\n  </component>\r\n  <component name=\"ProjectRootManager\" version=\"2\" languageLevel=\"JDK_26\" project-jdk-name=\"openjdk-26\" project-jdk-type=\"JavaSDK\">\r\n    <output url=\"file://$PROJECT_DIR$/out\" />\r\n  </component>\r\n</project>
===================================================================
diff --git a/.idea/misc.xml b/.idea/misc.xml
--- a/.idea/misc.xml	(revision 5545da7a06708f1e7e7645cde94668d10ecb52c7)
+++ b/.idea/misc.xml	(date 1782902087856)
@@ -8,7 +8,5 @@
       </list>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_26" project-jdk-name="openjdk-26" project-jdk-type="JavaSDK">
-    <output url="file://$PROJECT_DIR$/out" />
-  </component>
+  <component name="ProjectRootManager" version="2" project-jdk-name="openjdk-26" project-jdk-type="JavaSDK" />
 </project>
\ No newline at end of file
Index: src/main/java/com/xhl/C01_Java_Core/Java核心基础.md
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+># Java核心基础\r\n\r\n##  内容\r\n    类加载机制、HashMap 原理（重写 hashCode/equals）、锁机制（synchronized vs CAS）、\r\n    JVM（OOM 场景与调优）、Spring 循环依赖、MyBatis 代理模式、MySQL 事务隔离级别、线程池拒绝策略等。\r\n[技术评估表&&项目.xlsx](../../../../../../docs/StudyResource/%E6%8A%80%E6%9C%AF%E8%AF%84%E4%BC%B0%E8%A1%A8%26%26%E9%A1%B9%E7%9B%AE.xlsx)\r\n## 
===================================================================
diff --git a/src/main/java/com/xhl/C01_Java_Core/Java核心基础.md b/src/main/java/com/xhl/C01_Java_Core/Java核心基础.md
--- a/src/main/java/com/xhl/C01_Java_Core/Java核心基础.md	(revision 5545da7a06708f1e7e7645cde94668d10ecb52c7)
+++ b/src/main/java/com/xhl/C01_Java_Core/Java核心基础.md	(date 1782902087154)
@@ -3,5 +3,5 @@
 ##  内容
     类加载机制、HashMap 原理（重写 hashCode/equals）、锁机制（synchronized vs CAS）、
     JVM（OOM 场景与调优）、Spring 循环依赖、MyBatis 代理模式、MySQL 事务隔离级别、线程池拒绝策略等。
-[技术评估表&&项目.xlsx](../../../../../../docs/StudyResource/%E6%8A%80%E6%9C%AF%E8%AF%84%E4%BC%B0%E8%A1%A8%26%26%E9%A1%B9%E7%9B%AE.xlsx)
+
 ## 
\ No newline at end of file
